public/js/report/controller.js

```diff
@@ -391,6 +392,11 @@ angular.module('app').controller('reportCtrl', function ($scope, connection, $co
         });
     };
 
+    $scope.$on('repaint', function (event, args) {
+        $scope.selectedReport.properties.maxValue = $scope.selectedGaugeMax.value;
```

Comment:
Have you tried binding directly the text input to `selectedReport.properties.maxValue` ?